examples/meta/src/kernel/weighted_degree_position_string.sg.in

```diff
@@ -0,0 +1,25 @@
+File traindat = read_csv("@SHOGUN_DATA@/fm_train_dna.dat")
+File testdat = read_csv("@SHOGUN_DATA@/fm_test_dna.dat")
+
+#![create_features]
+Features feats_train = create_char_string_features(traindat, enum EAlphabet.DNA)
+Features feats_test = create_char_string_features(testdat, enum EAlphabet.DNA)
+#![create_features]
+
+#![create_kernel]
+int degree = 20
+WeightedDegreePositionStringKernel kernel(feats_train, feats_train, degree)
+kernel.init(feats_train, feats_train)
+IntVector shifts([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1])
```

Comment:
a few questions:

* do we really need 20 here?
* what about having a few different shifts/weights here? Increases coverage maybe?

Reply:
I guess you need as many values as vectors in feature? I am not sure how this is meant to be used though..?

Reply:
I mean we can leave this for now?

Reply:
yes, works with me